lib/experimental/Navigation/Carousel/index.stories.tsx

```diff
@@ -85,9 +118,59 @@ export const SneakPeek: Story = {
       </div>
     ),
   ],
+  argTypes: {
+    autoplay: { control: "boolean" },
+    showDots: { control: "boolean" },
+    showArrows: { control: "boolean" },
+    showPeek: { control: "boolean" },
+  },
   args: {
     showArrows: false,
     children: SLIDES,
     showPeek: true,
   },
 }
+
+const randomClasses = ["h-64", "h-full", "h-32", "w-32", "w-full", "w-64"]
+
+const SLIDES_RANDOM = [
+  <BarChartWidget
+    key="widget"
+    chart={BarChartStory.args as BarChartProps}
+    fullHeight
+  />,
+  ...Array.from({ length: 6 }, (_, i) => {
+    const randomHeight = randomClasses[Math.floor(i / 2)]
+    const randomWidth = randomClasses[Math.floor(i / 2) + 3]
+    return (
+      <Placeholder key={i + 1} className={`${randomHeight} ${randomWidth}`}>
+        Slide {i + 1} ({randomHeight} {randomWidth})
+      </Placeholder>
+    )
+  }),
+]
+
+/**
+ * Care using this component for distinct size widgets.
+ */
+export const MultipleWidths: Story = {
+  decorators: [
+    (Story) => (
+      <div className="h-full w-full p-6">
+        <Story />
+      </div>
+    ),
+  ],
+  argTypes: {
+    autoplay: { control: "boolean" },
+    showDots: { control: "boolean" },
+    showArrows: { control: "boolean" },
+    showPeek: { control: "boolean" },
+  },
+  args: {
+    showArrows: true,
+    children: SLIDES_RANDOM,
+    showPeek: true,
+    showDots: false,
```

Comment:
Do we really want to have all these props? What's the use case where we want to show arrows or not? Can't we just figure that out automatically? Same for `showPeek` or `showDots`.

Every time we add more choices to the users we create more potential bugs and inconsistencies. We need to learn from our mistakes and keep the interfaces as small as possible.